app/api/users.py

```
@@ -0,0 +1,54 @@
+from fastapi import APIRouter, Depends, HTTPException, status
+from sqlalchemy.ext.asyncio import AsyncSession
+from fastapi.security import OAuth2PasswordBearer, OAuth2PasswordRequestForm
+from app.core.database import get_db
+from app.schemas import UserCreate, UserResponse, Token
+from app.services import user_service
+from typing import Annotated
+
+router = APIRouter(prefix="/api/users", tags=["users"])
+oauth2_scheme = OAuth2PasswordBearer(tokenUrl="/api/users/login")
+
+@router.post("/register", response_model=UserResponse, status_code=status.HTTP_201_CREATED)
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)